KeyViewLoops.md

```diff
@@ -0,0 +1,45 @@
+# Key View Loops
+
+### Convention
+It is recommended to let the window automatically calculate your key view loop for you. There are rare cases in which the automatically calculated key view loop is not sufficient.
+
+### Rationale
```

Comment:
Both NSTableView and NSSegmentedControl are actually interesting examples. They both have focus to the whole control, but then use keyboard navigation to select the items within the control. In effect, this makes their behavior the same as a "FocusZone" on Fluent Web / win32. "nextKeyView" would not work for both cases because that only responds to tab and not arrow keys. Perhaps I could mention that in a section like "What if I want to use arrow keys to navigate" (Answer: Use an NSTableView)

Reply:
Well that is a common reason people think they need a custom key view loop, right? That might be the crux of my feedback. It's difficult to tell from this page what might be a valid case for using manual key view loops and using automatic key view loop and getting at least a few specific case studies on that would help.

Reply:
I added a section called "Alternatives" where I recommend people use an NSTableView or NSCollectionView if it fits their needs. I'm happy to change wording and whatnot. 

NSTableView/NSCollectionView were the only examples I could think of a common use case where Apple provides a good control to use. If people have other suggestions, I'll take them too.

Reply:
link to nscollectionview sample app in developer.apple.com?

Reply:
Or maybe just the developer page for NSCollectionView? https://developer.apple.com/documentation/appkit/nscollectionview

Reply:
also, can you talk about going to system preference > keyboard > input> turning on the tab checkbox to get things working? common things people from other platform do not understand.

Reply:
That might make this page more of a "Handling Keyboard Focus" instead of "Modifying the Key View Loop", which I think is fine, I just have to think how to fit the info properly. Also maybe I'd want to add screenshots

Reply:
Rewrote things a little more general. Would be nice to have eyes on it again just to make sure it still feels write for this guide.